javascript-source/commands/customCommands.js

```diff
@@ -441,6 +441,21 @@
             }
         }
 
+        if (message.match(/\(repeat\s[1-9]\d*,[\w\s]+\)/g)) {
+            var matches = message.match(/\(repeat\s([1-9]\d*),([\w\s]+)\)/);
+            var allMatch = matches[0];
+            var counter = matches[1];
+            var iteratingText = matches[2];
+
+            var tmpArray = [];
+
+            for(var i = 0; i < counter; i++) {
```

Comment:
The variable `counter` is never parsed as an `number`, meaning this could cause issues for people.

Reply:
If necessary, I can do it. But the group ([1-9]\d*) actually ensure that there are only positive integers allowed

Reply:
You're still comparing a number to a string doing `i < counter`, it may work fine on your system but can cause issues on other systems.